release.sh

```diff
@@ -186,6 +195,7 @@ if [ -z "$IMPORTED_KEY" ]; then
   echo "Failed to import GPG key"
   exit 1
 fi
+IMPORTED_PUBLIC_KEY="$(gpg_import "$RELEASE_GPG_PUBLIC_KEY_PATH")"
```

Comment:
@marko-bekhta Won't this fail on any other project where the env variable is *not* set? If so you just broke the release process of Search/Validator/others. Fortunately not ORM/Reactive since they don't use this script AFAIK.

Reply:
Ohh, I thought I had it as "conditional" ... but in the end it turned out to be useless, so I'll revert this change with the next update. We are looking with Andrea into hibernate-models and that would require some updates to the release scripts (so will include it there) 🙈 sorry